### PR TITLE
Optionally add breadcrumbs to category headings

### PR DESCRIPTION
**Pull request by Danwhy**
# Description

# Checklist

Pull request contains:

- [ ] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [ ] PR description includes description of change
- [ ] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.
- [ ] If component change, version updated

Screen Shot 2020-03-11 at 15 18 17
Screen Shot 2020-03-11 at 15 18 11

### CODE REVIEW

File: src/elements/category/package.json [hunk 1]
```diff
@@ -15,5 +15,8 @@
   "peerDependencies": {
     "@emotion/core": "^10.0.27",
     "react": "^16.7.0"
+  },
+  "dependencies": {
```
Comment (callumacrae) [inline src/elements/category/package.json]: This should be a devDependency, right?

Comment (Danwhy) [reply src/elements/category/package.json]: Good call, yeah

File: src/elements/category/src/index.tsx [hunk 2]
```diff
@@ -16,17 +17,34 @@ const DefaultContainer: React.FC = ({ children }) => (
...
+  const { theme }: any = useThemeUI()
+  const breadcrumbsVariant = theme.categoryTitle?.breadcrumbs?.variant
+
   return (
     <div
       sx={{
         paddingX: ['sm', 'md'],
-        paddingY: ['md', 'lg'],
+        paddingBottom: ['md', 'lg'],
+        paddingTop: ['xs', 'sm'],
```
Comment (callumacrae) [inline src/elements/category/src/index.tsx]: Does this reduce the padding when breadcrumbs aren't specified?

Comment (Danwhy) [reply src/elements/category/src/index.tsx]: Ah yeah, I will fix that

**Pull request merged**